### PR TITLE
Reindex folders

### PR DESCRIPTION
**Pull request by basiclines**
### Features

This PR is based on syncing the taxomomy from Mozilla wiki in to the Building Blocks final code.
The taxonomy has been approved from UX teams, both Mozilla and Telefónica
### Important

This PR is the base for the next several PR that we are going to do this week, so this should be merged first and as soon as possible :D

@rudylu @timdream and @jmcanterafonseca please, could you review it?

### CODE REVIEW

**Comment by basiclines** [comment]
I need some feedback from you guys about the PR @etiennesegonzac @jcarpenter @vingtetun Someone could accept this PR or move it forward?
Thx!

File: menus-dialogues/required.css [hunk 1]
```diff
@@ -0,0 +1,196 @@
+/* ----------------------------------
```
Comment (etiennesegonzac) [inline menus-dialogues/required.css:1]: This needs to be de-indentize like the other css files.

File: other/drawer/style.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
+/* ----------------------------------
+ * DRAWERS
```
Comment (etiennesegonzac) [inline other/drawer/style.css:2]: idem

File: other/headers/style.css [hunk 3]
```diff
@@ -5,15 +5,26 @@
 body[role="application"] section[role="region"] > header {
```
Comment (etiennesegonzac) [inline other/headers/style.css:5]: idem

File: other/lists/index.html [hunk 4]
```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
```
Comment (etiennesegonzac) [inline other/lists/index.html:1]: Doesn't look like you're using two spaces soft tabs here.

File: other/tabs/style.css [hunk 5]
```diff
@@ -0,0 +1,107 @@
+/* ----------------------------------
```
Comment (etiennesegonzac) [inline other/tabs/style.css:1]: de-indentation needed.

File: transitions/headers.css [hunk 6]
```diff
@@ -0,0 +1,206 @@
+/* ----------------------------------
```
Comment (etiennesegonzac) [inline transitions/headers.css:1]: idem

**Comment by etiennesegonzac** [comment]
@fabi1cazenave I would love if you could take a quick look at the markup here.

**Comment by basiclines** [comment]
Ok great i will take a look today!
thx @etiennesegonzac

**Comment by basiclines** [comment]
@etiennesegonzac Yesterday i was de-indenting all the css files in develop branch (for all the BB), please accept this pull request as the indentation problem will be solved with the last PR that i want to do (Final update with develop)

You can confirm the de-indent here: https://github.com/telefonicaid/Gaia-UI-Building-Blocks/tree/develop

**Comment by etiennesegonzac** [comment]
I'm not sure I understand.

If you want to "start fresh" with another full PR there is no issue. We can just close this one and accept the other one.

**Comment by basiclines** [comment]
Well the problem for the other one is that it has all the new code for all the BB, doing it in this way we can review it in small pieces and then make the final one with the latests fixes.
What do you prefeer? for me is ok to do a full PR with all the BB

Thx!

**Comment by etiennesegonzac** [comment]
Ideally it would have been in small pieces. But given that even the pieces are thousands lines of code any way, let's do it all at once.

Your code looks really solid so I'm not too worried :)

**Comment by basiclines** [comment]
Ok great so will cancel all the PR and i will make the big one!
Thank you!

**Pull request closed without merging**